### ISSUE TITLE
Remove jQuery and Bootstrap legacy assets

### ISSUE THREAD

**Issue by okms** [Puzzlepart/did#328]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
We are bundling jQuery and Bootstrap assets but only a very few elements require it. Lets remove this legacy cruft. This includes re-writing the profile dialog popup which today uses jQuery and Twitter Bootstrap css and js. 

![image](https://user-images.githubusercontent.com/170147/81291833-d607f400-906a-11ea-918d-9bdaf20cd4a3.png)

### Todos

- [x] Remove jQuery
- [x] Remove Bootstrap
- [x] Check all usage of bootstrap classes and replace with our own
- [x] Remove all uses of `col` and `row` 
- [x] Use React state for toggling menu
- [x] Fix toggle glitch (see **1**)
- [x] Fix customer dropdown in New Project form (see **2**)
- [x] Dropdown in resolve project modal should go out of the modal and not expand it (see **3**)

#### 1
![image](https://user-images.githubusercontent.com/7606007/81318462-95ba6d00-908e-11ea-8f62-2332f94e6ef8.png)

#### 2
![image](https://user-images.githubusercontent.com/7606007/81335351-c65ad080-90a7-11ea-9173-38f3a04d2f08.png)

#### 3
![image](https://user-images.githubusercontent.com/7606007/81335490-fe621380-90a7-11ea-82f2-7e9ccbde3017.png)
